tests/yam/agama/validate_headless.pm

```diff
@@ -0,0 +1,39 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Agama headless validataion
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+use testapi qw(
+  get_var
+  script_run
+  script_output
+  assert_script_run
+  upload_logs
+  select_console
+  console
+);
+use utils qw(systemctl);
+
+sub run {
+    select_console 'root-console';
+
+    # verify we are in muti-user and not graphic target.
+    assert_script_run('systemctl list-units --type=target | grep multi-user.target');
+    systemctl 'is-active graphical.target', expect_false => 1;
+    systemctl 'is-active graphical.target', expect_false => 1;
+    systemctl 'is-active x11-autologin.service', expect_false => 1;
+    systemctl 'is-enabled multi-user.target', expect_false => 0;
```

Comment:
why you add the `expect_false` shouldn't be the default here?

Reply:
removed.